.github/workflows/external.yml

```diff
@@ -0,0 +1,70 @@
+name: External-CI
+
+on: 
+  pull_request:
+
+jobs:
+  industrial_ci_external:
+    strategy:
+      matrix:
+        include:
+          - repo: 'ros-industrial/industrial_core'
+            ref: 'kinetic-devel'
+            env: {ROS_DISTRO: kinetic}
+
+          - repo: 'ros-industrial/motoman_experimental'
+            ref: 'kinetic-devel'
+            env: {ROS_DISTRO: kinetic, UPSTREAM_WORKSPACE: '.travis.rosinstall -ros-industrial/industrial_experimental/IRC_v2', ROS_REPO: ros}
+
+          - repo: 'ipa320/cob_calibration_data'
+            ref: 'indigo_dev'
+            env: {ROS_DISTRO: kinetic, ROS_REPO: ros, UPSTREAM_WORKSPACE: file, AFTER_SCRIPT: 'rosenv sh .travis.xacro_test.sh'}
+          
+          # - repo: 'ros/actionlib'
+          #   ref: '38ce66e2ae2ec9c19cf12ab22d57a8134a9285be'
+          #   depth: 0  # All history will be cloned
+          #   env: {ROS_DISTRO: indigo, ROS_REPO: ros, ABICHECK_URL: url, ABICHECK_MERGE: true} # actual URL will not be used in the case
```

Comment:
I didn't manage to get the merge ABICHECK to work. @ipa-mdl as I am not familiar with this could you help me have a look >.<
[log](https://github.com/Briancbn/industrial_ci/runs/1565708036?check_suite_focus=true)

Reply:
ABICHECK has some special handling for Travis, I will have a look